### DISCUSSION TITLE
[v5] Replace emotion with lighter (s)css modules

### DISCUSSION THREAD

**Discussion by LeunensMichiel** [JedWatson/react-select#4858] [Ideas]
Loving the library, but I noticed that for a select component, it is quite heavy. Looking on [bundlephobia](https://bundlephobia.com/package/react-select@5.1.0) I see that Emotion is responsible for a 15% increase in bundle size. For people using styled components or css modules, this is an entire extra styling library in their node_modules.

So my question: does the project depend heavily on emotion? If not, could it be replaced by much lighter classic css in a future PR? If so, I'd like to contribute.

**Comment by Methuselah96** [comment]
We are hoping and planning to make an unstyled export of `react-select` and allow the user to use whichever CSS solution they desire. We agree that the forced dependency on Emotion is undesirable. Thanks for bringing it up!

**Comment by ebonow** [comment]
Yup. Plus one.

**Comment by LeunensMichiel** [comment]
Cheers! Nice that to see it's in the pipeline. Congrats on the V5 release by the way!
